bmds_server/jobrunner/views.py

```diff
@@ -40,6 +40,7 @@ def get_redirect_url(self, *args, **kwargs):
 
 class JobDetail(DetailView):
     model = models.Job
+    print("job detail class")
```

Comment:
remove

Reply:
it might just be this change. there's some other css and style changes, are those intentional?

Reply:
removed. some of the css was updated.